drivers/mbox/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ zephyr_library_sources_ifdef(CONFIG_MBOX_NXP_MAILBOX   mbox_nxp_mailbox.c)
 zephyr_library_sources_ifdef(CONFIG_MBOX_ANDES_PLIC_SW   mbox_andes_plic_sw.c)
```

Comment:
commit description for bb585c08cc5f0040a441d1bc5a0eafafafc01b16 is misleading, existence of `interrupts` property is irrelevant

Reply:
Description changed.